src/main/java/use_case/translateText/TranslateTextInteractor.java

```diff
@@ -1,19 +1,19 @@
 package use_case.translateText;
 
 import entity.TextTranslator;
-import interface_adapter.translateText.TranslateTextOutputBoundary;
+import interface_adapter.translateText.TranslateTextPresenter;
 
 /**
  * The TranslateText Interactor.
  */
-public class TranslateTextInteractor {
```

Comment:
also for the the implementation of execute in the interactor, you should first assign the input text, input language, and output language before calling translate, otherwise you wouldn't have anything to translate